### DISCUSSION TITLE
Windows Relocation and Vertical Slider

### DISCUSSION THREAD

**Discussion by williamntv** [emoacht/Monitorian#416] [General]
Hello,

We would like to subscribe and use this app, and we want to know if the following features are available or can be made available:
1. Window can be moved, maximized, minimized and docked freely in a different location on another display.
2. Graphical slider can be resized freely to make it larger for touch screen use.
3. Graphical slider can be rearranged vertically to accommodate a small display's portrait mode.
4. Multiple monitors have their name simplified to Display 1, Display 2, etc.
5. Secondary display's slider can be hidden so users will only have access to brightness control for the main monitor.

We sincerely hope some or all of these features can be implemented. Extra compensation can be brought into discussion once we can try out a demo.

Thank you

**Comment by emoacht** [comment]
> Window can be moved, maximized, minimized and docked freely in a different location on another display.

No.

> Graphical slider can be resized freely to make it larger for touch screen use.

No. But you can select large slider whose size generally fit to touch operation.

> Graphical slider can be rearranged vertically to accommodate a small display's portrait mode.

No.

> Multiple monitors have their name simplified to Display 1, Display 2, etc.

You can edit the monitor name shown in this app. See the first para of [Remarks](https://github.com/emoacht/Monitorian#remarks).

> Secondary display's slider can be hidden so users will only have access to brightness control for the main monitor.

You can use `/preclude` option to exclude a specific monitor from this app. See https://github.com/emoacht/Monitorian/issues/270#issuecomment-1094184823

**Comment by williamntv** [comment]
Thank you

**Comment by emoacht** [comment]
You are welcome. If you have a specific difficulty in your actual use case, please let me know.
